tools/get-alpine-pkg-source.sh

```diff
@@ -36,6 +37,7 @@ do      case "$o" in
         s)      outdir=$OPTARG;;
         u)      urlfile=$OPTARG;;
         g)      gitdir=$OPTARG;;
+        p)      prefix=$OPTARG;;
         [?])    >&2 echo "Usage: $0 [-v] [-s <outdir>] [-u <urlfile>] [-t <tag>]+ [-e <evedir>] [-g <gitdir>] [<version>]"
```

Comment:
Update usage here and 2x below.

Reply:
But not clear to me we need an additional argument.

Reply:
Ah, but you missed the point of it. Which means I probably should add it to the document.

There are two things we want this script (and the kernel one) to do:

1. Put the files into some directory. This could be handled with `-s /tmp/sources/kernel` or `-s /tmp/source -p kernel`
2. Report where each file is relative to the `-s` directory, so that it can be used as a manifest.

In the first case, the output will be:

```
alpine,foo-1.2.3,abcdef123456,foo-1.2.3.abcdef123456
```

In the second case, the output will be:

```
alpine,foo-1.2.3,abcdef123456,alpine/foo-1.2.3.abcdef123456
```

If you have one source directory where you are dumping lots of different sources (e.g. go, alpine and kernel), and you want the reports to be relative to the "root" of the dump place, but for each to have a subdirectory it goes in, then you can do (pseudo):

```
get-alpine -s /tmp/sources -p alpine
get-kernel -s /tmp/sources -p kernel
get-go -s /tmp/sources -p golang
```

Then the outputs look like:

```
alpine,foo-1.2.3,abcdef123456,alpine/foo-1.2.3.abcdef123456
golang,github.com/some/pkg@v1.2.3,v1.2.3,golang/github.com_some_pkg@v1.2.3
```

Essentially, `-s` means, "the root of where I will place things", and `-p` means, where in that root should I place my particular things. If you only have one thing, it doesn't matter, but if you will have lots, then it will.